### ISSUE TITLE
Returning code 400 

### ISSUE THREAD

**Issue by erturkdotgg** [yandricr/gpti-js#1]
Hello, all parameters are correct. I'm using module since 2-3 weeks. Suddenly stopped working. Giving code 400. Everything is correct. I didn't change any code.

**Comment by yandricr** [comment]
Yes, it had given this error on the server: EIO: i/o error, read
 but I already solved it

**Comment by erturkdotgg** [comment]
Resolved.

**Comment by urielexis64** [comment]
The issue is back :(
